mani_skill/utils/wrappers/flatten.py

```diff
@@ -50,7 +51,12 @@ def observation(self, observation: Dict):
         if self.include_rgb and not self.include_depth:
             ret["rgb"] = images
         elif self.include_rgb and self.include_depth:
-            ret["rgbd"] = images
+            # todo: might not be the best way or the right place to perform this
```

Comment:
remove the todo. I think this is fine. I normally myself wouldn't merge rgb and depth but this is just for compatibility with a lot of old code from MS2 that assumed "rgbd" as a key.